miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditServiceRecordController.java

```diff
@@ -81,6 +86,31 @@ public String getKey() {
   @Autowired
   private FilesManager filesManager;
 
+  @Value("${miso.timeCorrection.uiZone:}")
+  private String uiZone;
+
+  PropertyEditor timestampEditor = new PropertyEditorSupport() {
+    @Override
+    public void setAsText(String text) throws IllegalArgumentException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)